src/website/views/components/data-table/examples/portal/_complex.pug

```diff
@@ -1,53 +1,49 @@
 h3 Complex
 .example.-mb--3
   .-p--3
-    .chi-card.-portal.-bg--white.-bg-md--grey-15
-      .chi-card__header.-sm
-        .chi-card__title Title
```

Comment:
Why remove the **Title**? In version [5.28.0](https://assets.ctl.io/chi/5.28.0/components/data-table/?theme=Portal#complex) we already had the title so please bring back and in the other themes as well.

Include it only in the example because it is not needed in the code snippets as it is not part of the data table example but part of our documentation. Thoughts @jllr?

Reply:
@lumdmtr @josecarlosgonzalezv we must follow the same pattern that we had in the previous chi versions with complex data table.